### DISCUSSION TITLE
Temporary folder for saving files in Next.js API route

### DISCUSSION THREAD

**Discussion by luis-gustavoj** [vercel/community#818] [Help]
I'm working with a project where i have a Next.js API route that receives a image, and save to a directory then the photo is edited and returned to the user then the image is deleted from the folder.

Testing it locally its working fine but when I deployed it to vercel the images are never found. I saw some old posts saying about a tmp folder in vercel, but i didnt manage to find any resources on how to that.

Is someone familiar with it and can help me?

**Comment by amyegan** [comment]
Hi @luis-gustavoj! I think that this https://github.com/vercel/community/discussions/241#discussioncomment-363002 is what you're looking for. But other [File Storage](https://vercel.com/docs/concepts/solutions/file-storage) strategies are likely to work better in most cases.
